core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -117,4 +118,184 @@ public static void Turn(PlayerBase player) throws Exception {
             }
         }
     }
+
+    //Will be continuously called in the game loop.
+    public void UserInteraction(ArrayList<String> text, PlayerBase player)
+    {
+        if(text.get(0).equals("move ")) {
```

Comment:
Hey, here it's hot necessary to check what is the direction, just check if this direction is acceptable, and send it as an argument. You may have a list, and see if the direction is in this list then send it as an argument, if not then display what are the accepted directions and return( the player would enter again the input)